### PR TITLE
Apis for adding and displaying score

### PR DESCRIPTION
**Pull request by Anand8317**
- Used APIs for creating a game that returns a unique Id.
- Adding a score makes a call to API to store its value
- Refresh makes a call to API to get all the user and their scores

### CODE REVIEW

File: src/modules/display.js [hunk 1]
```diff
@@ -1,12 +1,16 @@
...
+    let scores = await fetch('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/Jun0BzNiB6z5IoA83KCo/scores/');
+    scores = await scores.json();
+    scores = scores.result;
+    scores.forEach((score) => {
+      scoresList.innerHTML += `
+      <li>${score.user}: ${score.score}</li>
+      `;
+    });
   });
 };
```
Comment (muneebulrehman) [inline src/modules/display.js]: Hey @Anand8317, great job with the application. However, there is this one issue that you need to fix. I have attached [this video here](https://www.loom.com/share/64f7e8dcce324c92b79050fb214589e7), you can go through it where I explained the issue in detail.

**Review by muneebulrehman — CHANGES_REQUESTED** [review]
Hi @Anand8317 , 

Good job so far! I see you put a lot of effort into this project. However, there is one issue that you still need to work on to go to the next project but you are almost there!

## Highlights
- No linter errors
- Descriptive PR
- Good docs

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/display.js (hunk 1 above)
Comment (Anand8317) [reply src/modules/display.js]: Thanks for explaining the issue in detail. Highly appreciate the help. I have fixed the rendering on the page load issue. Again, thanks.

**Review by leonardodiasb — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @Anand8317 🔥 

You worked hard in all the instructions that the previous Code Reviewers give you👍🏽 

Now you can merge this branch 🚀 

## Highlights

 - No linter errors ✔️ 
 - Descriptive PR ✔️ 
 - Good docs ✔️ 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**